src/object.rs

```diff
@@ -1,3 +1,4 @@
+use crate::buf::Buf;
```

Comment:
I don't think these imports in all the files are actually needed since `Buf` is reexported at the root and most files have `use super::*`.